src/localstack.js

```diff
@@ -210,12 +218,12 @@ function checkServices (services = []) {
 async function localstackReady (container) {
   const stream = await container.attach({ stream: true, stdout: true, stderr: true });
   return new Promise((resolve) => {
-    const logs = new TempWriteBuffer(resolve);
+    const logs = new TempWriteBuffer(resolve, container.id);
     container.modem.demuxStream(stream, logs, logs);
   });
 }
 
-async function getConnection ({ versionTag = 'latest', services } = {}) {
+async function getConnection ({ versionTag = '0.10.6', services } = {}) {
```

Comment:
This seems unrelated to the title. What was the motivation for this? Defaulting to 'latest' feels pretty natural for a lot of other systems (e.g. pulling a docker image with no version specified).

Reply:
Over the last month the latest image hasn't been stable.  It also didn't match any official release.  Using latest is dangerous as it can cause previously passing builds to fail, or even cause issues in production.  
`:latest` does not mean the latest version, just whatever was added to the repo.  This can be older versions, that may or may not be compatible with current requirements.

https://vsupalov.com/docker-latest-tag/
https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375
https://blog.container-solutions.com/tagging-docker-images-the-right-way

Reply:
I agree that using 'latest' is generally a bad idea - particularly for reproducibility reasons. However, if this hardcoded semi-arbitrary version doesn't get updated regularly, as a consumer, I might get bitten by hard to track down bugs (i.e. old version not supporting something) due to using a version I never explicitly specified. I suspect I might would assume the latest version was being used until I had reason to believe otherwise.

At any rate, perhaps documenting this in the README somewhere would be beneficial.

Reply:
Users of the library can specify their own version.  Either way brings risk, but at least if we pick a version we can know what the risks are, instead of getting random errors that could be impossible to track down using the latest tag. 

I will update the README